### PR TITLE
 Adds an interface for disableRegisterCheck

### PR DESCRIPTION
**Pull request by VBGAMER45**

### CODE REVIEW

File: Sources/ManageRegistration.php [hunk 1]
```diff
@@ -299,6 +299,7 @@ function ModifyRegistrationSettings($return_config = false)
 	$config_vars = array(
 		array('select', 'registration_method', array($txt['setting_registration_standard'], $txt['setting_registration_activate'], $txt['setting_registration_approval'], $txt['setting_registration_disabled'])),
 		array('check', 'send_welcomeEmail'),
+        array('check', 'disableRegisterCheck'),
```
Comment (live627) [inline Sources/ManageRegistration.php]: whitespace

Comment (VBGAMER45) [reply Sources/ManageRegistration.php]: Yeah....Tried to fix... You can edit if this change will be merged.

**Comment by albertlast** [comment]
Is a change of the upgrade files for pg and mysql not missing?

**Comment by VBGAMER45** [comment]
Added it. I think it added to this pull request. Still a mystery to me how that works...

**Comment by SychO9** [comment]
> Added it. I think it added to this pull request. Still a mystery to me how that works...

It's not that hard, if you ever want to me to show you the ropes, you can always hit me up on discord.

**Comment by VBGAMER45** [comment]
> 
> 
> > Added it. I think it added to this pull request. Still a mystery to me how that works...
> 
> It's not that hard, if you ever want to me to show you the ropes, you can always hit me up on discord.

I will probably will just been very busy. What was interesting about this was concerned how the commit that I pushed was going to get linked to the this pull request but it looks like github did that automatically.

**Comment by jdarwood007** [comment]
If the code is already doing a empty check, then we don't need to handle this in upgrade or installs.  Just to simplify things.

**Pull request merged**